src/server/Html.js

```diff
@@ -34,6 +35,18 @@ export default function Html({
         <meta property="description" content="Team transparency, made easy." />
         <style dangerouslySetInnerHTML={{__html: lookCSSToken}} id={lookConfig.styleElementId} />
         <title>{title}</title>
+        {/* segment.io analytics */}
+        {process.env.SEGMENT_WRITE_KEY &&
```

Comment:
Not sure i follow, We'll still be building and minifying the same thing for each visitor, right?